script.js

```diff
@@ -0,0 +1,112 @@
+const game = { turn: 0 };
+const players = [{ score: 0 }, { score: 0 }];
+const diagonals = [
+  [
+    { row: 0, column: 0 },
+    { row: 1, column: 1 },
+    { row: 2, column: 2 },
+  ],
+  [
+    { row: 0, column: 2 },
+    { row: 1, column: 1 },
+    { row: 2, column: 0 },
+  ],
+];
+let cells;
+
+function getCellCoordinates(cell) {
+  const index = cells.indexOf(cell);
+  return { row: Math.floor(index / 3), column: index % 3 };
+}
+
+function initializeBoard() {
+  game.board = [
+    [null, null, null],
+    [null, null, null],
+    [null, null, null],
+  ];
+  registerCellClickListeners();
+}
+
+function updateBoard(coordinates) {
```

Comment:
I agree that `updateBoard` isn't the best name, though I'm not a huge fan of `recordTurn`, either, as the very next function uses a different sense of "turn" ("a state-changing action" versus "the player who is to take the next turn"). One way around this would be to use "turn" for the first sense and "turnPlayer" for the second.

Reply:
I'm definitely on board with changing the property named `turn` to something else. Yesterday I almost suggested making it a char variable called `playerMark` that holds either `X` or `O`. However I would also be good with the easier option of a simple rename to `turnPlayer`